### ISSUE TITLE
W002: X-UA-Compatible check is brittle, not semantic

### ISSUE THREAD

**Issue by atz** [twbs/bootlint#394]
As per the [wiki](https://github.com/twbs/bootlint/wiki/W002), bootlint is looking for a meta tag:
```html
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

However, semantically it should also accept:
```html
<meta http-equiv="X-UA-Compatible" content="IE=edge,extra=4">
```
or similar statements that accomplish the same thing.  (In olden times that might have been `chrome=1`.  In future ones, it could be something else.)

**Comment by sideshowbarker** [comment]
The only value the HTML spec allows is `IE=edge`:

https://html.spec.whatwg.org/multipage/semantics.html#attr-meta-http-equiv-x-ua-compatible

> (In olden times that might have been chrome=1. In future ones, it could be something else.)

As far as I know nobody has any plans to ever add anything more for `X-UA-Compatible`.

The HTML spec previously said any use at all of `X-UA-Compatible` was an outright error, regardless of the value. We changed the spec to allow `IE=edge` because that seems to be the only value which at this point it still makes any sense at all for anybody to ever use.

**Comment by cvrebert** [comment]
Closing as invalid.
